### ISSUE TITLE
Clarification of customisation of input controls

### ISSUE THREAD

**Issue by matatk** [w3c/xaur#44]
In [section 3.6](https://raw.githack.com/w3c/apa/38a0617945cdd79579be632add45e78d0b78e39c/xaur/index.html#customisation-of-control-inputs) I'm not clear on the first sentence. I mean it seems to stand alone. (The point of the section as a whole that this is an important requirement is an important one, of course.)

**This needs to have the XAUR label, but I don't have permissions to make that change on this repo.** 

*[Original feedback email, for reference](https://lists.w3.org/Archives/Public/public-apa-admin/2020Jan/0021.html)*

**Comment by RealJoshue108** [comment]
Thanks @matatk The sentence could be phrased better in order to define the problem? So maybe a problem statement would be better. Something like:

">>The user may not have the ability<< to modify their input preference or use a variety of input devices. The remapping of keys used to control movement or interaction in virtual environments is not currently required by WCAG. It is nevertheless noted in the literature as desirable"

Would this make it clearer?

**Comment by matatk** [comment]
Yes, @RealJoshue108, I think that would be good. I have some additional info too...

There's a great article on [remapping in the Game Accessibility Guidelines](http://gameaccessibilityguidelines.com/allow-controls-to-be-remapped-reconfigured/) which explains why this is important and, crucially, why it's much better to do this at the application level than system/OS level.

(To paraphrase: if you do it at application/game level, then you can provide much more natural and helpful remapping facilities. Games may have a menu system, and then several different modes of play, such as driving or walking around. Remapping provided by the game can cover all/either of these modes, as appropriate, but system-level remapping can't take them into consideration.)

So the system-level remapping is still a good thing to have, just in case, but per-application (or game) remapping is _really_ important.

This sounds similar to w3c/xaur#34, and it may make sense to merge the two, but I'm not sure yet (more reading required).
